beacon-chain/sync/subscriber.go

```diff
@@ -119,7 +119,7 @@ func (r *RegularSync) subscribe(topic string, validate validator, handle subHand
 		}
 
 		if !validate(ctx, msg, r.p2p, fromSelf) {
-			log.WithField("message", msg.String()).Debug("Message did not verify")
+			//log.WithField("message", msg.String()).Debug("Message did not verify")
```

Comment:
why comment out instead of delete?

Reply:
oops

Reply:
I think we should delete it, it was annoying